.github/workflows/reusable_monitoring.yml

```diff
@@ -40,8 +40,8 @@ permissions:
   contents: read
 
 env:
-  UPLOADED_LOG_NAME: checkpoint
-  LOG_FILE: checkpoint_log.txt
+  UPLOADED_LOG_NAME: ct_checkpoint
```

Comment:
whoops, I think we flipped these! These should remain the same, and we should change the other.